index.md

```diff
@@ -4,49 +4,210 @@ title: Software Engineering for Busy Experts
-Some more text.
+Email me at [hi@tkiley.co.uk](mailto:hi@tkiley.co.uk) to discuss how this course can help your company develop software effectively.
 
 ## The Course
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec laoreet magna justo, eleifend sagittis magna pellentesque a. Vivamus nec porttitor tellus. Sed vitae volutpat diam. Nunc blandit mauris quis arcu pretium, sit amet hendrerit sapien molestie. Integer lobortis urna nec dignissim varius. Pellentesque at nunc vitae velit varius sodales at tincidunt elit. Vivamus mi sapien, tempor id lectus porta, laoreet feugiat eros. Cras vel dapibus risus, ut commodo ex. Nulla in auctor elit. Donec non placerat tellus, vitae pulvinar est. Ut eget tincidunt nisl. Curabitur congue libero eget tortor congue, laoreet gravida sem iaculis.
+The course is designed to cover the key areas of modern software engineering best practices. Each seminar will help developers to adopt _industry standard approaches_ to developing software.
+
+The course consists of 5 seminars:
+
+ 1. [Source Control](#source-control)
+ 2. [Automated Tests](#automated-tests)
+ 3. [“Good” Code](#good-code)
+ 4. [Code Review](#code-review)
+ 5. [Continuous Integration](#continuous-integration)
+
+## About Thomas
+
+I have been writing code for ten years in a range of industries including science, technology, games and healthcare. In this time I have learnt _the importance of good software practices_ in writing correct code quickly, and in collaboration with others.
+
+In my time as team lead I was able to see how good software practices can _speed up delivery_ and increased quality.
+
+I am really motivated to use my experience of software engineering experts fulfil their technologies potential. Please [get in touch](mailto:hi@tkiley.co.uk) to unlock your expertise.
+
+## Course Syllabus
+
+The course consists of 5 seminars:
 
  1. [Source Control](#source-control)
  2. [Automated Tests](#automated-tests)
  3. [“Good” Code](#good-code)
  4. [Code Review](#code-review)
- 5. [Continuous Integration (CI)](#continuous-integration-ci)
+ 5. [Continuous Integration](#continuous-integration)
+
+Each of the seminars features _practical examples_ to see how these techniques can be applied. These seminars take the form of a 45 minute presentation and time for questions. This interactivity ensures the techniques can be applied in your domain.
+
+The seminars can be delivered in person at your office, via a video call or a hybrid approach.
+
+In addition to the off-the-shelf course, I am happy to tailor the course to _match your needs, experience and technologies_.
+
+**[Get in touch](mailto:hi@tkiley.co.uk) today to craft a bespoke course for your business.**
+
+I also like to foster an on-going relationship after you have completed the course to _help you put these tools and techniques into practice_.
+
+[Get in touch](mailto:hi@tkiley.co.uk) today to learn the skills needed for _effective software development_.
 
+### Source Control
```

Comment:
"Seminar 1: Source Control"

I'd forgotten that we were in a specific context by the time we get to this heading so add a reminder?